openpype/hosts/houdini/api/creator_node_shelves.py

```diff
@@ -173,6 +173,7 @@ def install():
         os.remove(filepath)
 
     icon = get_openpype_icon_filepath()
+    tab_menu_label = os.environ.get("AVALON_LABEL") or "AYON"
```

Comment:
Can we also apply this fallback [here](https://github.com/ynput/OpenPype/pull/5850/files#diff-9c740d20922834f0dd17be15a68e41a056376e797a404ba7a4b1d4f9cb0788beR7)?

Reply:
And while we're at it maybe also in the `add_self_publish_button` logic just so throughout Houdini it'll at least be consistent?